checks.d/mysql.py

```diff
@@ -200,7 +200,12 @@ def _collect_metrics(self, host, db, tags, options, queries):
         # Max of 20 queries allowed
         if isinstance(queries, list):
             for index, check in enumerate(queries[:self.MAX_CUSTOM_QUERIES]):
-                self._collect_dict(check['type'], {check['field']: check['metric']}, check['query'], db, tags=tags)
+                if 'tags' in check:
+                    total_tags = tags + check['tags']
+                else:
+                    total_tags = tags
+
```

Comment:
you could probably just do it all in a single line:
`total_tags = tags + check.get('tags', [])`

check.get('tags', []) will return en empty list if there's no tags in check, or checks['tags'] if it is... So it shortens it a bit.

As you said, here it comes down to readability vs space efficiency.